src/style.css

```diff
@@ -1,3 +1,89 @@
-body {
+:root {
+  --main-color: #00acdc;
+}
+
+ul,
+ol {
+  padding: 0;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style elements using `.class`. In your case, you could create a class like `.list-reset` and apply it to `ul` or `ol` where necessary.